webtool/gradle/common/src/commonMain/kotlin/Pilot.kt

```diff
@@ -7,4 +7,78 @@ import kotlin.js.JsExport
 @JsExport
 data class Pilot(
     val name: String,
+    val numHouseholds: int,
+    val households: List<Households>,
+    val numCompanies: int,
+    val companies: List<Company>,
+    val numSolarfarms: int,
+    val solarfarms: List<Solarfarm>,
+    val numWindfarms: int,
+    val windfarms: List<Windfarm>,
+    val numBatteries: int,
+    val batteries: List<Battery>,
+    val numHeatStorages: int,
```

Comment:
Het aantal los opslaan is niet nodig, dit kunnen we opvragen via `heatStorages.size` (6x)